src/utils/validation.js

```diff
@@ -0,0 +1,21 @@
+export const validateEmail = (email) => {
+  if (!email) {
+    return 'Email is required';
+  }
+
+  const emailRegex = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailRegex.test(email)) {
+    return 'Email is not valid';
+  }
+};
```

Comment:
The function 'validateEmail' does not return anything if the email is valid. Consider returning a value indicating success or null/undefined to signify no error.